bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -388,7 +344,18 @@ def _should_retry(exc):
     return reason == 'backendError' or reason == 'rateLimitExceeded'
 
 
-def get_sub_prop(container, keys, default=None):
+DEFAULT_RETRY = retry.Retry(predicate=_should_retry)
```

Comment:
I wonder if we should move this to a public module? Currently these docs won't get rendered.

Reply:
See PR #5552.